pom.xml

```diff
@@ -95,7 +95,7 @@
         <version.org.codehaus.woodstox.woodstox-core>5.0.3</version.org.codehaus.woodstox.woodstox-core>
         <version.org.fusesource.jansi>1.11</version.org.fusesource.jansi>
         <version.org.jboss.aesh>0.66.11</version.org.jboss.aesh>
-        <version.org.jboss.byteman>3.0.3</version.org.jboss.byteman>
+        <version.org.jboss.byteman>4.0.0-ALPHA</version.org.jboss.byteman>
```

Comment:
I have no idea but I guess once JDK9 is final? Try ask Andrew Dinn.
If you see this commit problematic I can exclude it from change set too.
There's already Byteman BETA0 available, but it works just on JDK9 based on jake branch.
JDK9 mainline isn't supported yet. ALPHA works on both JDK9 versions.